src/SymptomHistory/index.tsx

```diff
@@ -26,9 +49,24 @@ const SymptomHistory: FunctionComponent = () => {
         contentContainerStyle={style.contentContainer}
         alwaysBounceVertical={false}
       >
-        <Text style={style.headerText}>
-          {t("symptom_history.symptom_history")}
-        </Text>
+        <View style={style.headerContainer}>
+          <Text style={style.headerText}>
+            {t("symptom_history.symptom_history")}
+          </Text>
+
```

Comment:
I think we want this to be at the bottom and have the same style as the `ExposureHistory` screen and button.